spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/influx/InfluxDbAutoConfigurationTests.java

```diff
@@ -69,4 +89,14 @@ private void load(String... environment) {
 		this.context = ctx;
 	}
 
+	@Configuration
+	static class Config {
```

Comment:
I'd rename that class so that it's more explicit what it simulates. `CustomerBuilderConfig` for instance.